src/expression.c

```diff
@@ -9764,6 +9764,20 @@ Expression *SliceExp::syntaxCopy()
     return se;
 }
 
+static bool isOpDollar(VarExp* ve)
+{
+    VarDeclaration* decl = ve->var->isVarDeclaration();
+    if (decl)
+    {
+        if (decl->ident->len == 8 &&
+            memcmp(decl->ident->string, "__dollar", decl->ident->len) == 0) // TODO is there a function for this?
+        {
+            return true;
+        }
+    }
+    return false;
+}
```

Comment:
You're right. I'll wait and see if any more logic has to be added :) If not I'll change it.